### DISCUSSION TITLE
Compare backtest results with other package

### DISCUSSION THREAD

**Discussion by graceyangfan** [nautechsystems/nautilus_trader#722] [Show and tell]
Compared nautilus_trader ema_cross stratgy with vnpy(https://github.com/vnpy/vnpy)
symbol: ETH-USDT  data_time:2022-6-1 to 2022-7-1  trade_time:2022-6-2 to 2022-7-1
commision_rate:0.0004  fill_rate =1 slippage=0 
(1)nautilus trader version: latest develop 
https://github.com/graceyangfan/nautilus_tutorial/blob/main/compare/nautilus/nautilus_trader.png
(2)vnpy 
https://github.com/graceyangfan/nautilus_tutorial/blob/main/compare/vnpy/vnpy_result.jpeg

stratgy and other process files are in https://github.com/graceyangfan/nautilus_tutorial/tree/main/compare
would like to see other package backtest results

**Comment by cjdsellers** [comment]
Thanks for providing this @graceyangfan 

I think for a reasonable comparison, we'd need the other platform to also be event driven rather than vectorized. There would still be many variables to nail down even from that point

**Reply by graceyangfan** [reply]
vnpy is event based, the biggest difference might be the order filled criterion,it perform directly on bars,I will do another test，because nautilus_trader have fixed several bugs，and my understandimg of total equal account money of nautilus_trader is wrong，it should be account money plus unrealized pnl @cjdsellers

**Reply by cjdsellers** [reply]
OK, understood. Your work here is very much appreciated! Please continue to share your findings :pray:
